### PR TITLE
User can see article images

### PR DESCRIPTION
**Pull request by designerofthing**
[PT Link](https://www.pivotaltracker.com/story/show/173002016)
Chore
```  
As a user
In order to see article visuals
I want to see articles images on main page and article page
```
- remove placeholders and add image (article.image)
- Force image to take a specific size on the page
- Change fixtures to include fake image URLs

dnsimages

### CODE REVIEW

**Review by kermit-klein — COMMENTED** [review]
We gotta check Erik's highlighted background for title

**Comment by erikbjoern** [comment]
What do we see on the screenshot? Two articles on top of each other?

**Comment by erikbjoern** [comment]
Check old tests, Semaphore fails!

**Comment by designerofthing** [comment]
> Check old tests, Semaphore fails! 
fixed!

> What do we see on the screenshot? Two articles on top of each other?
Main page article list one article after the other vertically with images instead of placeholders like the  story.

> We gotta check Erik's highlighted background for title
??

**Comment by erikbjoern** [comment]
Ah, are we going to have it without any margin? Looks good I think, though I'm not sure what's the best reading experience. What Ali is referring to is the screenshot I posted in the channel, with a grey overlay behind the title. But that's not needed to be implemented here either.

**Review by erikbjoern — APPROVED** [review]

**Review by jysmys — APPROVED** [review]

**Comment by kermit-klein** [comment]
I meeaaaan, the screenshot Erik sent out on channel about Image and Image Title

**Comment by designerofthing** [comment]
> Ah, are we going to have it without any margin? Looks good I think, though I'm not sure what's the best reading experience. What Ali is referring to is the screenshot I posted in the channel, with a grey overlay behind the title. But that's not needed to be implemented here either.

I thought about it and played around with various settings on the grid but went back to the story and saw it was out of scope.  I suggest we make another story for the layout, if we want to match the lo-fis or not.  

> I meeaaaan, the screenshot Erik sent out on channel about Image and Image Title

I see that now, it looks great, maybe we can implement it with the layout changes in a new story?

File: public/locales/sv/translation.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "Morning": "Morgon",
+  "Afternoon": "Eftermiddag",
```
Comment (ViaMarcus) [inline public/locales/sv/translation.json]: Out of scope

File: cypress/fixtures/article_list.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-    { "id": 1, "title": "title 1" },
-    { "id": 2, "title": "title 2" },
-    { "id": 3, "title": "title 3" },
-    { "id": 4, "title": "title 4" },
-    { "id": 5, "title": "title 5" }
+    { "id": 1, "title": "title 1",  "image": "https://picsum.photos/800/600" },
+    { "id": 2, "title": "title 2",  "image": "https://picsum.photos/500/400" },
+    { "id": 3, "title": "title 3",  "image": "https://picsum.photos/400/300" },
+    { "id": 4, "title": "title 4",  "image": "https://picsum.photos/700/500" },
+    { "id": 5, "title": "title 5",  "image": "https://picsum.photos/400/300" }
```
Comment (ViaMarcus) [inline cypress/fixtures/article_list.json:7]: These all scale to the same size when displayed?

File: src/components/ArticleCard.jsx [hunk 3]
```diff
@@ -1,24 +1,23 @@
 import React from "react";
-import { Placeholder, Grid } from "semantic-ui-react";
+import { Placeholder, Grid, Image } from "semantic-ui-react";
```
Comment (ViaMarcus) [inline src/components/ArticleCard.jsx]: Remove placeholder import

**Review by ViaMarcus — CHANGES_REQUESTED** [review]

File: public/locales/sv/translation.json (hunk 1 above)
Comment (designerofthing) [reply public/locales/sv/translation.json]: don't know how it jumped branches but I have deleted it now.

File: src/components/ArticleCard.jsx (hunk 3 above)
Comment (designerofthing) [reply src/components/ArticleCard.jsx]: done

**Pull request merged**